### PR TITLE
Add mxnet-operator v1 examples

### PR DESCRIPTION
**Pull request by Jeffwan**
1. Reorganize exmaple folder, since training codes and Docker doesn’t need to be changed for every operator version, we move `train` and `tune` to top example level.

2. Simplify train Dockerfile, we don’t have to build mxnet from scratch, mxnet/python:gpu already provides python binding.

3. Remove train_mnist.py and use one in incubator-mxnet.

4. Add v1 kubernetes manifest.

Signed-off-by: Jiaxin Shan <seedjeffwan@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,10 +106,10 @@ should be created for each replica.
...
+Before you use the auto-tuning example, there is some preparatory work need to be finished in advance. To let TVM tune your network, you should create a docker image which has TVM module. Then, you need a auto-tuning script to specify which network will be tuned and set the auto-tuning parameters, For more details, please see https://docs.tvm.ai/tutorials/autotvm/tune_relay_mobile_gpu.html#sphx-glr-tutorials-autotvm-tune-relay-mobile-gpu-py. Finally, you need a startup script to start the auto-tuning program. In fact, mxnet-operator will set all the parameters as environment variables and the startup script need to reed these variable and then transmit them to auto-tuning script. We provide an example under examples/tune/, tuning result will be saved in a log file like resnet-18.log in the example we gave. You can refer it for details.
```
Comment (terrytangyuan) [inline README.md:112]: Use markdown syntax for TVM reference?

File: examples/train/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM mxnet/python:gpu
+
+RUN apt-get update && \
+    apt-get install -y git && \
+    git clone https://github.com/apache/incubator-mxnet.git
```
Comment (terrytangyuan) [inline examples/train/Dockerfile:5]: Do we need to check out a particular release to make surer this always works?

File: README.md (hunk 1 above)
Comment (Jeffwan) [reply README.md:112]: Yeah. I will update it.

File: examples/train/Dockerfile (hunk 2 above)
Comment (Jeffwan) [reply examples/train/Dockerfile:5]: Our base image uses mxnet/python:gpu which should be the latest version?  code is from master. I am ok to pin to a stable version like 1.5.x or 1.6.x. I will make the change

**Comment by Jeffwan** [comment]
@terrytangyuan Address all comments, please have another look

File: README.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ kubectl create -f manifests/deployment.yaml
 You create a training job by defining a `MXJob` with `MXTrain` mode and then creating it with.
 
 ```
-kubectl create -f examples/v1beta1/train/mx_job_dist_gpu.yaml
+kubectl create -f examples/train/mx_job_dist_gpu_v1beta1.yaml
```
Comment (terrytangyuan) [inline README.md:61]: So the v1 manifests are not ready to be used in README?

**Pull request merged**